gradle.properties

```diff
@@ -3,4 +3,8 @@ org.gradle.daemon=true
 org.gradle.parallel=true
 org.gradle.jvmargs=-Xmx1024m
 
+bouncyCastleVersion=1.82
+flywayVersion=11.14.0
+springBootVersion=3.5.7
```

Comment:
If we make these reusable, we should use Version Catalogs: https://docs.gradle.org/current/userguide/version_catalogs.html

Reply:
I don't know. But since Version Catalogs are a first-class Gradle feature, I would expect these tools to implement support for that (if they haven't already).